projects/components/src/icon/icon.component.scss

```diff
@@ -1,5 +1,7 @@
 .ht-icon {
   display: flex;
+  justify-content: center;
```

Comment:
this is not generating some side effect with the current icons? I'm just double checking...

Reply:
I went through and looked at several pages and didn't see any issues. Not saying a hidden alignment issue might not pop up, but it will have no real impact and be easy to fix if we run across one.